helm/velero-app/templates/volumesnapshotlocation.yaml

```diff
@@ -3,6 +3,9 @@ apiVersion: velero.io/v1
 kind: VolumeSnapshotLocation
 metadata:
   name: {{ include "velero.volumeSnapshotLocation.name" . }}
+  annotations:
+    "helm.sh/hook": post-install
```

Comment:
We need this post-install otherwise Helm reported it could not find `VolumeSnapshotLocation` CR kind.